data/taylorjeandev.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Taylor Jean",
+  "url": "www.github.com/taylorjeandev",
+  "image": "",
```

Comment:
Kindly add a URL to your image

Reply:
You can add an image "URL" by just pasting the link to your Github profile on a browser and adding .png at the end.
Example: https://github.com/cheloghm.png